docs/internal/design/oobe/integration-review.html

```
@@ -0,0 +1,480 @@
+<title>IronClaw OOBE — Integration Review</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)